tasks/main.yml

```diff
@@ -18,13 +36,124 @@
     - rsyslog
     - files
 
-- name: "Set forwarding to logstash"
+- name: "Set forwarding to logstash (without TLS)"
   lineinfile: >
     dest=/etc/rsyslog.conf
     regexp="^\*\.\* @{{ hostvars[logstash_forwarder]['private_ip']|default(hostvars[logstash_forwarder]['ansible_ssh_host']) }}:{{ logstash_syslog_port|default('514') }}"
     line="*.* @{{ hostvars[logstash_forwarder]['private_ip']|default(hostvars[logstash_forwarder]['ansible_ssh_host']) }}:{{ logstash_syslog_port|default('514') }}"
     state=present
-  when: rsyslog_conf.stat.exists|bool and logstash_forwarder is defined
+  when: use_rsyslog_tls|bool == false and rsyslog_conf.stat.exists|bool and logstash_forwarder is defined
+  notify:
+    - Restart rsyslog
+  tags:
+    - rsyslog
+    - files
+
+- name: "Set forwarding (with TLS). Download papertrail-bundle.pem"
+  get_url: >
+    url='https://papertrailapp.com/tools/papertrail-bundle.pem'
+    dest="/etc/pki/tls/certs/papertrail-bundle.pem"
+    mode=0444
+  when: use_rsyslog_tls|bool == true and rsyslog_conf.stat.exists|bool and logstash_forwarder is defined
```

Comment:
there should be an `use_papertrail` flag here.
It's not because you are using TLS that you are using PT.